docs/development/repositories-structure.md

```diff
@@ -50,6 +50,8 @@ This repository provides a framework for writing Service Manager Service Broker
 
 * interfaces that the `Peripli/service-broker-proxy-cf` and `Peripli/service-broker-proxy-k8s` implement in order to talk to the  corresponding platform controller during state reconcilation.
 
+Tree structure look like this:
```

Comment:
"The tree structure looks like this:"

Also - if we add this sentence here, shouldn't we add it everywhere else that a tree structure is illustrated?

Reply:
Adopted changes from doc-fixes repo.